elements/lisk-chain/test/unit/process.spec.ts

```diff
@@ -745,6 +807,39 @@ describe('blocks/header', () => {
 				expect(storageStub.entities.Account.upsert).not.toHaveBeenCalled();
 			});
 
+			it('should add produced block for generator', async () => {
+				const generator = await stateStore.account.get(
+					getAddressFromPublicKey(block.generatorPublicKey),
+				);
+				expect(generator.producedBlocks).toEqual(1);
+			});
+
+			it('should update generator balance with rewards and fees - minFee', async () => {
+				const generator = await stateStore.account.get(
+					getAddressFromPublicKey(block.generatorPublicKey),
+				);
+				let expected = block.reward;
+				for (const tx of block.transactions) {
+					// TODO: substract minFee
```

Comment:
is it gonna be addressed in this PR? if not can we have here with issue

Reply:
Updated comment